### PR TITLE
[Course Task]: test-cloud-pkg-csidriver-uds

### PR DESCRIPTION
**Pull request by jxs1211**
Signed-off-by: xian-jie.shen <327411586@qq.com>

**What type of PR is this?**
ut

**What this PR does / why we need it**:
ut
[119](https://github.com/kubeedge/community/issues/119)

**Which issue(s) this PR fixes**:
none

**Special notes for your reviewer**:
```
go test uds_test.go uds.go utils.go -cover -coverprofile=cover.out && go tool cover -func=cover.out && go tool cover -html=cover.out -o coverage.html
ok      command-line-arguments  0.010s  coverage: 30.4% of statements
/home/going/workspace/kubeedge/cloud/pkg/csidriver/uds.go:37:           NewUnixDomainSocket             100.0%
/home/going/workspace/kubeedge/cloud/pkg/csidriver/uds.go:47:           Connect                         100.0%
/home/going/workspace/kubeedge/cloud/pkg/csidriver/uds.go:65:           Send                            90.0%
```
**Does this PR introduce a user-facing change?**:
none

### CODE REVIEW

File: cloud/pkg/csidriver/uds_test.go [hunk 1]
```diff
@@ -0,0 +1,161 @@
+/*
+Copyright 2019 The KubeEdge Authors.
```
[the remark this answers is not in the document]
Comment (jxs1211) [reply cloud/pkg/csidriver/uds_test.go]: done

File: cloud/pkg/csidriver/uds_test.go [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+			wantErr:  true,
+		},
+		{
+			name:     "dial error",
+			filename: "unix://" + f.Name(),
+			want:     nil,
+			wantErr:  true,
+		},
+	}
+	for _, tt := range tests {
```
Comment (vincentgoat) [inline cloud/pkg/csidriver/uds_test.go:84]: New a listener and then making a real connection would be better.

**Pull request closed without merging**